### PR TITLE
Missing dependencies

### PR DESCRIPTION
**Pull request by j-rivero**
During the packaging of the package I noted that we have some missing dependencies in the package. I've also include a little fix to use the catkin variable for installing.

I'm assuming that we have no intention to install the tests artifacts: ign_publisher and ros1_publisher.

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -7,11 +7,13 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <depend>geometry_msgs</depend>
   <depend>roscpp</depend>
+  <depend>rostest</depend>
```
Comment (chapulina) [inline package.xml]: I think this can be `<test_depend>`

**Comment by caguero** [comment]
+1

File: package.xml (hunk 1 above)
Comment (j-rivero) [reply package.xml]: fixed aa25d2667f6c14f9c94f2f03b824a2d04051dee5

**Comment by caguero** [comment]
@j-rivero , feel free to resolve conflicts and merge.

**Pull request merged**